<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToroidWindow</class>
 <widget class="QMainWindow" name="ToroidWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Toroid Designer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout2">
    <item>
     <widget class="QGroupBox" name="typeGroupBox">
      <property name="title">
       <string>Type</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout4">
       <item>
        <widget class="QRadioButton" name="powderedIronButton">
         <property name="text">
          <string>&amp;Powdered Iron</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ferriteButton">
         <property name="text">
          <string>F&amp;errite</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="parametersGroupBox">
      <property name="title">
       <string>Parameters</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="coreTypeLabel">
         <property name="text">
          <string>&amp;Core type:</string>
         </property>
         <property name="buddy">
          <cstring>coreTypeComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QComboBox" name="coreTypeComboBox"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="coreMaterialLabel">
         <property name="text">
          <string>Core &amp;material:</string>
         </property>
         <property name="buddy">
          <cstring>coreMaterialComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QComboBox" name="coreMaterialComboBox"/>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="inductanceLabel">
         <property name="text">
          <string>&amp;Desired inductance:</string>
         </property>
         <property name="buddy">
          <cstring>inductanceSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="inductanceSpinBox">
         <property name="prefix">
          <string/>
         </property>
         <property name="suffix">
          <string> uH</string>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="designInformationGroupBox">
      <property name="title">
       <string>Design Information</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="designInductanceLabel">
         <property name="text">
          <string>Calculated inductance: ?.? uH</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="designCoreTypeLabel">
         <property name="text">
          <string>Core type: Powdered Iron T-?-? (?)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="designTurnsLabel">
         <property name="text">
          <string>Turns: ??</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="designWireGaugeLabel">
         <property name="text">
          <string>Maximum wire gauge: ??</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="designFreqRangeLabel">
         <property name="text">
          <string>Frequency range: ?? - ?? MHz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="designMuLabel">
         <property name="text">
          <string>Mu factor: ?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="designAlLabel">
         <property name="text">
          <string>Al: ?? Î¼H/100 turns</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>288</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About Toroid Designer</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
